### PR TITLE
mc: new maintainer & fix build error

### PR DESCRIPTION
**Pull request by electimon**
Read: https://k1ss.org/style
Read: https://k1ss.org/guidestones

---

## Description of package

Self-explanatory.

## Existing package

- [*] I am the maintainer of this package.

### CODE REVIEW

File: community/mc/build [hunk 1]
```diff
@@ -6,5 +6,7 @@
     --sysconfdir=/etc \
     --disable-nls
 
+patch -p1 < glib.patch
+
```
Comment (dylanaraps) [inline community/mc/build]: I'd prefer the patch as close to the start of the file as possible. :+1:

Otherwise, LGTM.

**Comment by dylanaraps** [comment]
Ping @electimon

**Comment by electimon** [comment]
> Ping @electimon

Sorry just saw this today, when I get back from school I'll fix it

**Comment by dylanaraps** [comment]
Ping @electimon

**Comment by dylanaraps** [comment]
Removed `mc` from Community. Open another PR when you're ready.

**Comment by dylanaraps** [comment]
Removed `mc` from Community. Open another PR when you're ready.

**Pull request closed without merging**